pages/buildkite_compute/overview.md

```diff
@@ -0,0 +1,82 @@
+
+You can migrate existing pipelines to compute services, to learn more see [Compute Pipeline Migration](/docs/buildkite-compute/pipeline-migration)
+
+## Coming soon
+
+### API support for hosted queues
+We are working on adding functionality in the API to allow configuration of hosted queues.
+
+### macOS image configuration in the UI
+We are building the ability to choose the software versions you require to be installed on the MacOS instances used in your queues.
+
+### Buildkite secrets
+
+Only in the rarest cases does CI not need to access outside services, and in these cases, the usability of the CI is severely limited. To use CI effectively - and to move toward CD, continuous deployment - your CI system needs to be able to safely and securely interact with outside services like observability platforms, cloud providers, and other services.
+
+To do this, you need to be able to securely store secrets like API credentials, SSH keys, and other sensitive information, and be able to use them safely and effectively in your builds. Buildkite Secrets provides a way to do this - we'll securely store your secrets, and provide a way for you to access them in your builds.
+
+Buildkite Secrets are an encrypted key-value store, where secrets are available to your builds via the Buildkite Agent. Secrets are encrypted both at rest and in transit using SSL, and are decrypted server-side when accessed by the agent. The agent makes it easy to use these secrets in your build scripts, and provides a way to inject secrets into your build steps as environment variables.
+
+Secrets will initially be scoped per-cluster - that is, agents outside of the cluster the secret belongs to will not be able to access that secret. We have additional work on the roadmap to allow secrets to be scoped per-pipeline.
+
+Until Buildkite secrets are available and if you would like to continue using your third party secrets provider like AWS SSM, GC Secrets or Hashicorp Vault we provide plugins that allow you to access these services. If a plugin for the service you use is not listed below please contact support.
+
+<table>
+    <thead>
+        <tr><th>Service</th><th>Plugin</th></tr>
+    </thead>
+    <tbody>
+        <tr><td>AWS SSM</td><td><a href="https://github.com/buildkite-plugins/aws-assume-role-with-web-identity-buildkite-plugin">aws-assume-role-with-web-identity-buildkite-plugin</a></td></tr>
+        <tr><td>GC Secrets</td><td><a href="https://github.com/buildkite-plugins/gcp-workload-identity-federation-buildkite-plugin">gcp-workload-identity-federation-buildkite-plugin</a></td></tr>
+        <tr><td>Hashicorp Vault</td><td><a href="https://github.com/buildkite-plugins/vault-secrets-buildkite-plugin">vault-secrets-buildkite-plugin</a></td></tr>
+    </tbody>
+</table>
+
+
+
+### Ability to SSH into a machine
+
+We are working on allowing direct SSH access into the compute instances.
+
+### Usage metrics
+
+Enhanced usage metrics across your compute queues.
+
+
+
```

Comment:
```suggestion
Enhanced usage metrics across your hosted agents' queues.
```

What is the purpose of this section? It currently doesn't seem to serve a purpose...